Losant/losant-react-experience
# [TL-200] Fixing build / deploy instructions + additional updates

- Changed `.env.local` to `.env.development` (see https://create-react-app.dev/docs/adding-custom-environment-variables#what-other-env-files-can-be-used)
- Additional `.gitignore` entries
- Several package upgrades
- Fix deploy instructions (`/static` directory)
- Adding public "About" page + simple test (removed App test as it was failing)
- Additional README updates

To test:

1. Create a new Losant application.
2. Import the [React template](https://app.losant.com/applications/recent/template-library/search?search=react&templateId=636d6136fe22dcc03d4fd188).
3. Create an experience user you can sign in with for testing.
4. Clone this repo to your local computer and check out this branch.
5. Follow the steps in the README exactly.
6. Verify you can run the project locally.
7. Verify it builds correctly.
8. Verify you can deploy it to your Losant application files.
9. Verify the deployed bundle works in your end user experience.
